### PR TITLE
fix: Editing makefile to add stateless CNI binary to the archives

### PR DESCRIPTION
**Pull request by behzad-mir**
**Issue Fixed**:
 Editing makefile to add stateless CNI binary to the archives

- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

**Review by tamilmani1989 — DISMISSED** [review]

File: dropgz/build/windows.Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN tdnf install -y unzip
 RUN tdnf upgrade -y && tdnf install -y ca-certificates
 
 FROM tar AS azure-vnet
-ARG AZCNI_VERSION=v1.5.28
+ARG AZCNI_VERSION=v1.6.0
```
Comment (jpayne3506) [inline dropgz/build/windows.Dockerfile:9]: With this do we need to make a `release/v1.5` branch now @rbtr?

Comment (rbtr) [reply dropgz/build/windows.Dockerfile:9]: Yes

Comment (behzad-mir) [reply dropgz/build/windows.Dockerfile:9]: I'm making a release right after the merging the PR. If you need to make a 1.5 release then let me know If I need to remove this tag?

Comment (behzad-mir) [reply dropgz/build/windows.Dockerfile]: Changed it back to 5.28. Will make another PR for 1.6 dropGZ

**Review by paulyufan2 — APPROVED** [review]

**Review by paulyufan2 — APPROVED** [review]

**Pull request merged**